### ISSUE TITLE
Show error when an invalid feature is added

### ISSUE THREAD

**Issue by SaiintBrisson** [serayuzgur/crates#124]
**Is your feature request related to a problem? Please describe.**
Sometimes a typo is committed when declaring a feature, and I don't get feedback until I try compiling the project.

**Describe the solution you'd like**
It would be helpful to add an error on the feature declaration line, or perhaps below the feature itself.

**Comment by SaiintBrisson** [comment]
After giving it a look, I figured that it might not be possible, and therefore I will close this issue

**Comment by WilliamVenner** [comment]
@SaiintBrisson I think this is actually possible, you can reopen this

**Comment by WilliamVenner** [comment]
Well, it's possible, but after a review of the parsing code it would take a lot of work and a huge refactor. Might not be worth it.

**Comment by SaiintBrisson** [comment]
Yeah, that's what I thought too, I'll leave this open for now, the maintainers may close this if they want to.

**Comment by serayuzgur** [comment]
@saiintbrisson  if you describe how to check it, I can take a look at the parser and see how much time it will take.

**Comment by lvkv** [comment]
This would be a great enhancement. I think it'd also work well with crate feature listing and/or feature autocomplete. Does this already exist in the Crates extension?
